jib-core/src/test/java/com/google/cloud/tools/jib/api/JibContainerBuilderTest.java

```diff
@@ -127,14 +135,9 @@ public void testToBuildConfiguration()
             .setEventHandlers(new EventHandlers().add(mockJibEventConsumer));
 
     JibContainerBuilder jibContainerBuilder =
-        Jib.from(baseImage)
+        new JibContainerBuilder(baseImage, spyBuildConfigurationBuilder)
             .setLayers(Arrays.asList(mockLayerConfiguration1, mockLayerConfiguration2));
-    BuildConfiguration buildConfiguration =
-        jibContainerBuilder.toBuildConfiguration(spyBuildConfigurationBuilder, containerizer);
-
-    Assert.assertEquals(
```

Comment:
I think we should still have this check for checking the `ContainerConfiguration` built by `JibContainerBuilder` is the one used in the `BuildConfiguration` it generated.